src/mechanics/cell/CellIpData.h

```diff
@@ -12,11 +12,11 @@ namespace NuTo
 class CellIpData
 {
 public:
-    CellIpData(const ElementCollection& elements, NuTo::Jacobian jacobian,
-               NaturalCoords ipCoords)
+    CellIpData(const ElementCollection& elements, NuTo::Jacobian jacobian, NaturalCoords ipCoords, int ipNum)
         : mElements(elements)
         , mJacobian(std::move(jacobian))
         , mIPCoords(std::move(ipCoords))
+        , mIPNum(ipNum)
```

Comment:
1) Naming. We decided to write things like `Ip or Pde or Iga` instead of `IP, PDE, IGA` - same goes for `CellID`
2) On the first glance, I thought _Why do we need the number of integration points here?_ Maybe rename that to `IpId`?

Reply:
1. my apologies
2. probably better

Reply:
Just another remark:

My git blame tells me, that the line above the marked line is from some TTitscher. Please remind him of point 1. `mIPCoords` --> `mIpCoords` :stuck_out_tongue:

Reply:
Hmm, who should remind him? Ill make an issue :smile_cat: